File: mycollab-dao/src/main/scala/com/mycollab/db/arguments/SearchRequest.scala
package com.mycollab.db.arguments

import scala.beans.BeanProperty

/**
  * @author MyCollab Ltd
  * @since 5.3.5
  */
@SerialVersionUID(1L)
class SearchRequest(@BeanProperty var currentPage: Integer, @BeanProperty var numberOfItems: Integer) {
  @BeanProperty val requestedUser: String = GroupIdProvider.getRequestedUser
}

object SearchRequest {
  val DEFAULT_NUMBER_SEARCH_ITEMS = 25
}

File: mycollab-dao/src/main/java/com/mycollab/db/arguments/SearchRequest.kt
package com.mycollab.db.arguments

/**
 * @author MyCollab Ltd
 * @since 6.0.0
 */
open class SearchRequest(var currentPage: Int, var numberOfItems: Int, requestedUser: String = GroupIdProvider.requestedUser) {
    companion object {
        @JvmField val DEFAULT_NUMBER_SEARCH_ITEMS = 25
    }
}